code/game/objects/items/weapons/grenades/flashbang.dm

```diff
@@ -102,3 +102,10 @@
 	M.stats.addTempStat(STAT_BIO, stat_def, 10 SECONDS, "flashbang")
 	M.stats.addTempStat(STAT_MEC, stat_def, 10 SECONDS, "flashbang")
 	M.update_icons()
+
+/obj/item/weapon/grenade/flashbang/nt
+	name = "NT FBG \"Holy Light\""
+	desc = "A \"NanoTrasen\" flashbang grenade."
```

Comment:
I wouldn't think nanotrasen would make holy granades, so i'd reccomend something like
`An old \"NanoTrasen\" flashbang granade, modified to spread the light of god.`

Reply:
Done